src/main/java/Time/WeekView.java

```diff
@@ -33,4 +37,69 @@ public void previousWeek() {
     public LocalDate getStartOfWeek() {
         return startOfWeek;
     }
+
+    public void printMonthView(TaskManager taskManager) {
+        YearMonth yearMonth = YearMonth.from(startOfWeek);
+        LocalDate firstOfMonth = startOfWeek.withDayOfMonth(1);
+        LocalDate lastOfMonth = yearMonth.atEndOfMonth();
+        LocalDate current = firstOfMonth.with(TemporalAdjusters.previousOrSame(DayOfWeek.SUNDAY));
```

Comment:
Innovative use of the TemporalAdjusters package to standardize the day of week the month starts with on the calendar view, similar to an actual calendar!